radicle-cli/examples/rad-node.md

```diff
@@ -0,0 +1,99 @@
+The radicle node is our daemon friend that is running as a background
+process. It allows us to interact with the network as well as storing
+some key data that we may be interested in.
+
+<details>
+<summary>🚧 `rad node start` is under construction 🚧</summary>
+If the node is not running we can start it by using the `rad node
+start` command:
+
+<!-- ``` -->
+<!-- $ rad node start -->
+<!-- ``` -->
+</details>
+
+We can confirm the status of the node at any time by using the `rad
+node status` command (or just `rad node` for short):
+
+```
+$ rad node status
+✓ The node is running
+```
+
+The node also allows us to connect with other nodes in the
+network. For example, let's connect to a known peer using their Node
+ID and their network Address:
+
+```
+$ rad node connect z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk 0.0.0.0:3679
+✓ Connecting to node...
+✓ Successfully connected to z6Mkt67…v4N1tRk @ 0.0.0.0:3679
+```
+
+The node also allows us to query data that it has access too such as
+the tracking relationships and the routing table. Before we explore
+those commands we'll first track a peer so that we have something to
+see.
+
+```
+$ rad track z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --alias Bob
+Establishing 🌱 tracking relationship for heartwood
+
+✓ Tracking relationship with Bob (z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk) established
+! Warning: fetch after track is not yet supported
+```
+
+Now, when we use the `rad node tracking` command we will see
+information for repositories that we track -- in this case a
+repository that was already created, and other nodes that we track:
+
+```
+$ rad node tracking
```

Comment:
I think it's a little weird to output two tables; especially once they start being a bit bigger it won't be super useful. I'd suggest we make `--repos` or `--nodes` required, or have it default to repos if no flag is passed.

Reply:
The `rad node *` commands are meant to be a little more on the "plumbing" side btw, so it's ok to be more rigid here.

Reply:
Ok, that makes sense :+1: